src/test/java/com/gooddata/featureflag/ProjectFeatureFlagTest.java

```diff
@@ -52,4 +60,33 @@ public void shouldVerifyEquals() throws Exception {
                 .withIgnoredFields("links")
                 .verify();
     }
+
+    @Test
+    public void changeProjectFeatureFlagTest() {
```

Comment:
Please read https://github.com/gooddata/gooddata-java/wiki/Testing first. This should not be in this unit test, but in acceptance tests (see `FeatureFlagServiceAT`) since it's dependent on GD environment (API).

Reply:
Thanks for commend, I rewrote it to  ` FeatureFlagServiceAT `